src/test/java/bio/terra/common/ValidationUtilsTest.java

```diff
@@ -1,31 +1,58 @@
 package bio.terra.common;
 
 import bio.terra.common.category.Unit;
+import org.apache.commons.lang3.RandomStringUtils;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
-import org.junit.runner.RunWith;
-import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-
-@RunWith(SpringRunner.class)
-@SpringBootTest
-@AutoConfigureMockMvc
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
To be consistent with the rest of our code, this should be 
```suggestion
import static org.junit.Assert.assertThat;
```

Reply:
This might be a separate PR: we do actually use assertJ in other places

Reply:
Why would this be done in a separate PR? This is the PR that adds this import to this file. You can see that the junit version is the more commonly used form (40 imports vs 7 imports). Why add another import to the version that is not the commonly used form in our codebase? That just adds another import that will need to be cleaned up later.